composer.json

```diff
@@ -26,7 +26,8 @@
         "php-http/promise": "^1.0",
         "psr/http-message-implementation": "^1.0",
         "psr/log": "^1.0",
-        "psr/simple-cache": "^1.0"
+        "psr/simple-cache": "^1.0",
+        "ext-json": "*"
```

Comment:
Why adding this in the main `composer.json` file ?
I only found the use of the `json_decode`. Since PHP 5.2.0, JSON extension is included by default.